src/main/java/io/gingersnapproject/cdc/cache/hotrod/HotRodService.java

```diff
@@ -30,48 +27,60 @@
 import io.gingersnapproject.cdc.translation.ColumnStringTranslator;
 import io.gingersnapproject.cdc.translation.IdentityTranslator;
 import io.gingersnapproject.cdc.translation.JsonTranslator;
-import io.quarkus.arc.lookup.LookupIfProperty;
+import io.quarkus.runtime.ShutdownEvent;
+import io.quarkus.runtime.StartupEvent;
 
-@LookupIfProperty(name = "service.hotrod.enabled", stringValue = "true", lookupIfMissing = true)
 @Singleton
 public class HotRodService implements CacheService {
 
    @Inject NotificationManager eventing;
 
    @Inject Configuration config;
 
-   ConcurrentMap<URI, RuleCacheManager> otherURIs = new ConcurrentHashMap<>();
-   private static final String OFFSET_CACHE_NAME = "debezium-offset";
-   private static final String SCHEMA_CACHE_NAME = "debezium-schema";
+   volatile RemoteCacheManager rcm;
 
-   @Override
-   public boolean supportsURI(URI uri) {
-      return uri.getScheme().startsWith("hotrod");
+   public void start(@Observes StartupEvent e) {
+      rcm = new RemoteCacheManager(config.cache().uri());
+      createCaches();
+      eventing.backendStartedEvent();
    }
 
+   public void shutdown(@Observes ShutdownEvent e) {
+      if (rcm != null) {
+         rcm.stopAsync().whenComplete((ignore, t) -> {
+            if (t != null) {
+               eventing.backendFailedEvent(t);
+            } else {
+               eventing.backendStoppedEvent();
+            }
+         });
```

Comment:
I am not 100% on the lifecycle of quarkus, but maybe set rcm to null here as well.